docs/schema.md

```diff
@@ -174,7 +180,13 @@ Developers can make a schema as abstract schema specifying type=abstract. The de
   - `lock_related` - locks the resource and related resources
   - `skip_related` - locks the resource but leaves related resources unlocked
   - (empty): default, no locking
-    
+
+- retry_create_on_duplicate_entry (boolean)
+
+  Whether create logic should be retried in case of "Duplicate Entry" on DB. Default to false.
+  Useful when pre_create logic is setting some generated data on column where unique index was applied.
```

Comment:
@p-kozlowski , agreed. It would be better to allow extensible error handler with with some flow control capabilities (retry, return, etc). With single flag `retry_create_on_duplicate_entry`, we can expect many more.